### PR TITLE
[9.0] [MIG] html_image_url_extractor module

### PR DESCRIPTION
**Pull request by cubells**
* Updated README.rst

@Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: html_image_url_extractor/models/ir_fields_converter.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        :param str html_content:
+            HTML contents from where to extract the images.
+
+        :param int limit:
+            Only get up to this number of images.
+
+        :param bool fail:
+            If ``True``, exceptions will be raised.
+        """
+        # Parse HTML
```
Comment (lasley) [inline html_image_url_extractor/models/ir_fields_converter.py:30]: Parsing HTML with Regex is never really the way to go, please see [this famous answer](https://stackoverflow.com/questions/1732348/regex-match-open-tags-except-xhtml-self-contained-tags/1732454#1732454) on SO for an amazing elaboration.

Why not just `xml.dom.minidom.parseString(html).getElementsByTagName('img')` + other logic?

Comment (pedrobaeza) [reply html_image_url_extractor/models/ir_fields_converter.py:30]: This is only a migration of something done on v8. As this extract utility is very specific, I wouldn't touch this too much...

**Review by lasley — CHANGES_REQUESTED** [review]
Thanks @cubells but I'm 👎  on this, please see my inline comment regarding the use of Regexes for this purpose.

File: html_image_url_extractor/models/ir_fields_converter.py (hunk 1 above)
Comment (lasley) [reply html_image_url_extractor/models/ir_fields_converter.py:30]: But when we bring something into OCA aren't we guaranteeing that it works by OCA standards? This definitely does not work in all instances - I've been burned by this before.

Comment (pedrobaeza) [reply html_image_url_extractor/models/ir_fields_converter.py:30]: Why is not working? It's in v8 and used by many modules without problems

Comment (lasley) [reply html_image_url_extractor/models/ir_fields_converter.py:30]: Alright wait maybe I'm misinterpreting something here. The regex is being applied against an element node, so we're good. My bad I misread there.

**Review by lasley — APPROVED** [review]
In light of inline convo, this was a stupid me situation. LGTM! 👍

**Pull request merged**